### PR TITLE
Add href attribute to pf-sort links

### LINKED ISSUE #213: pf-sort directive not accessible
The links in the sort menu are missing `href` attributes and can't be activated using the keyboard. They also don't get the pointer mouse icon on hover because of the missing `href`.

/cc @jwforres @rhamilto

### PR DESCRIPTION
**Pull request by spadgett**
Fixes #213

/cc @jwforres @rhamilto

### CODE REVIEW

**Comment by jeff-phillips-18** [comment]
You can try using href="javascript:void(0);" rather than href="" to get the tests to pass.

**Comment by spadgett** [comment]
@jeff-phillips-18 I can make the change, but I'm unable to run the tests because of this error:

```
PhantomJS 1.9.8 (Mac OS X) pf-select Page with pf-select directive should respond to changes in ng-options FAILED
	Expected 4 to be 3.
	Error: Expected 4 to be 3.
	    at /Users/sam/git/angular-patternfly/test/select/select.spec.js:54
PhantomJS 1.9.8 (Mac OS X) pf-select Page with pf-select directive should respond to ng-model changes FAILED
	Expected 4 to be 3.
	Error: Expected 4 to be 3.
	    at /Users/sam/git/angular-patternfly/test/select/select.spec.js:83
	Expected '' to be 'a'.
	Error: Expected '' to be 'a'.
	    at /Users/sam/git/angular-patternfly/test/select/select.spec.js:88
```

**Comment by spadgett** [comment]
Updated.

**Comment by jeff-phillips-18** [comment]
I don't see any issues with the pf-select tests.

**Comment by spadgett** [comment]
> I don't see any issues with the pf-select tests.

It could be environmental, but they consistently fail for me.

File: dist/angular-patternfly.js [hunk 1]
```diff
@@ -1595,7 +1595,7 @@ angular.module('patternfly.charts').directive('pfLineChart', ["pfUtils", functio
          */
         $scope.getLineData = function (chartData) {
           var lineData  = {
-            type: 'line'
+            type: $scope.setAreaChart ? "area" : "line"
           };
```
Comment (jeff-phillips-18) [inline dist/angular-patternfly.js]: Looks like you need to rebuild

Comment (spadgett) [inline dist/angular-patternfly.js]: I tried, the diff won't disappear :(

**Comment by jeff-phillips-18** [comment]
You might try cleaning out the node_modules and lib directories and redoing npm install and bower install

**Comment by spadgett** [comment]
> You might try cleaning out the node_modules and lib directories and redoing npm install and bower install

Thanks, that fixed the test error. Still seeing the diff after `grunt build`... Any ideas?

**Comment by jeff-phillips-18** [comment]
I see... it is a problem with the previous commit not rebuilding :( 
Next commit will fix it

LGTM

**Comment by spadgett** [comment]
Rebased.

**Pull request merged**